doc/api/timers.md

```diff
@@ -2,87 +2,139 @@
+an immediate timer is queued from inside an executing callback, that timer will
+not be triggered until the next event loop iteration.
+
+If `callback` is not a function, an [`Error`][] will be thrown.
+
+### setInterval(callback, delay[, arg][, ...])
+
+* `callback` {Function} The function to call when the timer elapses.
+* `delay` {number} The number of milliseconds to wait before calling the
+  `callback`.
+* `[, arg][, ...]` Optional arguments to pass when the `callback` is called.
 
 Schedules repeated execution of `callback` every `delay` milliseconds.
-Returns a `intervalObject` for possible use with [`clearInterval`][]. Additional
-optional arguments may be passed to the callback.
+Returns a `Timeout` object for possible use with [`clearInterval`][].
+
+When `delay` is larger than `2147483647` or less than `1`, the `delay` will be
+set to `1`.
 
-To follow browser behavior, when using delays larger than 2147483647
-milliseconds (approximately 25 days) or less than 1, Node.js will use 1 as the
-`delay`.
+If `callback` is not a function, an [`Error`][] will be thrown.
 
-If `callback` is not a function `setInterval()` will throw immediately.
+### setTimeout(callback, delay[, arg][, ...])
 
-## setTimeout(callback, delay[, arg][, ...])
+* `callback` {Function} The function to call when the timer elapses.
+* `delay` {number} The number of milliseconds to wait before calling the
+  `callback`.
+* `[, arg][, ...]` Optional arguments to pass when the `callback` is called.
 
 Schedules execution of a one-time `callback` after `delay` milliseconds.
-Returns a `timeoutObject` for possible use with [`clearTimeout`][]. Additional
-optional arguments may be passed to the callback.
+Returns a `Timeout` object for possible use with [`clearTimeout`][].
 
-The callback will likely not be invoked in precisely `delay` milliseconds.
+The `callback` will likely not be invoked in precisely `delay` milliseconds.
 Node.js makes no guarantees about the exact timing of when callbacks will fire,
 nor of their ordering. The callback will be called as close as possible to the
 time specified.
 
-To follow browser behavior, when using delays larger than 2147483647
-milliseconds (approximately 25 days) or less than 1, the timeout is executed
-immediately, as if the `delay` was set to 1.
+When `delay` is larger than `2147483647` or less than `1`, the `delay` will be
+set to `1`.
+
+If `callback` is not a function, an [`Error`][] will be thrown.
+
+## Cancelling Timers
+
+The [`setImmediate`][], [`setInterval`][], and [`setTimeout`][] methods each
+return objects that represent the scheduled timers. These can be used to cancel
+the timer and prevent it from triggering.
+
+### clearImmediate(immediateObject)
+
+* `immediateObject` {Immediate} An immediate timer object as returned by
+  [`setImmediate`][].
+
+Cancels an `Immediate` object created by `setImmediate()`.
+
+### clearInterval(timeoutObject)
```

Comment:
The old form of `#timers_clearinterval_intervalobject` is currently referenced here and in `globals.md`, btw.